docs/collecting-data/collecting-from-own-applications/snowplow-tracker-protocol/example-requests/index.md

```diff
@@ -0,0 +1,383 @@
+---
+title: "Example Tracker Requests"
+date: "2022-10-10"
+sidebar_position: 100
+---
+
+# Example Tracker Requests
+
+Below are a number of example Tracker Protocol requests. All examples are POST requests with a JSON body.
+
+All requests are sent to `com.snowplowanalytics.snowplow/tp2` unless you have specified a custom POST path ([via Snowplow Console](/docs/using-the-snowplow-console/accessing-collector-configuration/index.md) or [Collector Configuration](/docs/pipeline-components-and-applications/stream-collector/configure/index.md)).
+
+All payloads should be wrapped in a `payload_data` Self Describing JSON.
+
+```json
+{
+  "schema": "iglu:com.snowplowanalytics.snowplow/payload_data/jsonschema/1-0-4",
+  "data": [
+    {
+      ...
+    }
+  ]
+}
+```
+
+## Self Describing Event
+
+This is an example of a _viewed_product_ event from the web (using the recommended Base64 encoding and the `ue_px` property):
+
+```json
+{
+  "schema": "iglu:com.snowplowanalytics.snowplow/payload_data/jsonschema/1-0-4",
+  "data": [
+    {
+      "e": "ue",
+      "ue_px": "eyJzY2hlbWEiOiJpZ2x1OmNvbS5zbm93cGxvd2FuYWx5dGljcy5zbm93cGxvdy91bnN0cnVjdF9ldmVudC9qc29uc2NoZW1hLzEtMC0wIiwiZGF0YSI6eyJzY2hlbWEiOiJpZ2x1OmNvbS5teV9jb21wYW55L3ZpZXdlZF9wcm9kdWN0L2pzb25zY2hlbWEvMS0wLTAiLCJkYXRhIjp7InByb2R1Y3RfaWQiOiJBU08wMTA0MyIsInByaWNlIjo0OS45NX19fQ==",
+      "eid": "5a305bf5-32fc-40d9-b0e8-66052f0f2c95",
+      "tv": "js-3.5.0",
+      "tna": "biz1",
+      "aid": "website",
+      "p": "web",
+      "cookie": "1",
+      "cs": "UTF-8",
+      "lang": "en-GB",
+      "res": "3440x1440",
+      "cd": "30",
+      "tz": "Europe/London",
+      "dtm": "1665398554151",
+      "vp": "693x1302",
+      "ds": "678x9015",
+      "vid": "29",
+      "sid": "be9520e7-16a5-4d4e-afa1-8e269f99a1cf",
+      "duid": "85094061-f702-4b62-a46d-20f7226b4741",
+      "url": "https://snowplow.io/",
+      "stm": "1665398554153"
+    }
+  ]
+}
+```
+
+The encoded base64 string above represents the following JSON:
```

Comment:
Above where? Assumes users know what base64 looks like. Is it `ue_px`? I would state that explicitly